eng/pipelines/ci.yml

```diff
@@ -99,6 +99,7 @@ jobs:
         - pwsh: |
             $currentVersion = node -p -e "require('./src/package.json').version";
             $currentVersion="$currentVersion-$(Build.BuildNumber)";
+            cd src
```

Comment:
Why are we doing this?

Reply:
To only publish relevant generator files?

Reply:
The package.json for this project is under the `src` directory.

Reply:
Ah got it. Thank you!